### PR TITLE
Add sastodeal.com

### PR DESCRIPTION
**Pull request by sodhancha**
Nepal's own E-commerce.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,6 +73,7 @@ If you cannot provide a valid website for your company your PR will not be accep
 - [Rigo Technology](https://www.rigonepal.com/) | Lazimpat, Kathmandu | Rigo provides Information Technology Consultancy, System Design and Payroll Outsourcing Service
 
 ## S - U
+- [Sastodeal | Be Genuine](https://www.sastodeal.com) | Tripureshwor, Kathmandu
```
Comment (mesaugat) [inline README.md]: Thank you for the PR. Can you also add a one line description of your company? 

And `Be genuine` should probably go in the description.

Comment (mesaugat) [inline README.md]: I am trying to make the list consistent as much as we can.

**Comment by sodhancha** [comment]
Added description. You can accept it now.

**Comment by mesaugat** [comment]
Thank you 👍

**Pull request merged**